pkg/types/validation/installconfig.go

```diff
@@ -1571,7 +1572,7 @@ func validateCredentialsNumber(controlPlane *types.MachinePool, fencing *types.F
 	if fencing != nil {
 		numOfCredentials = len(fencing.Credentials)
 	}
-	if numOfCpReplicas == 2 {
+	if numOfCpReplicas == 2 && !installConfig.IsArbiterEnabled() {
```

Comment:
Nit: IIUC basiclly we want to skip this validation for the Arbiter use case, so  I think it would be nicer to move installConfig.IsArbiterEnabled() to the first check where the validation is skipped.
```
if controlPlane == nil || controlPlane.Replicas == nil || installConfig.IsArbiterEnabled()  {
		// invalid use case covered by a different validation.
		return errs
}
```